Release-Notes-R4-draft.md

```diff
@@ -14,19 +14,22 @@ Open Source Software nicely combines with
 [Sovereign Cloud Stack](https://scs.community/) to form a
 rather complete set of tools that can be used to provide
 Gaia-X conforming services on top of sovereign infrastructure.
+In addition, we have knowledge of deployments were > 100 clusters
+are being managed using the capi-based SCS reference implementation
+for k8s cluster management.
```

Comment:
This is a great update! I'm wondering if we could share some details, e.g.: approx. number of VCPUs, memory, and CSP name that running this beast :)

Reply:
Enrico @flyersa told this to me - not sure how much he can say publicly.